packages/sanity/src/core/form/inputs/DateInputs/base/calendar/Calendar.tsx

```diff
@@ -202,7 +202,6 @@ export const Calendar = forwardRef(function Calendar(
         <Flex>
           <Box flex={1}>
             <CalendarMonthSelect
-              moveFocusedDate={moveFocusedDate}
               onChange={handleFocusedMonthChange}
               labels={{
```

Comment:
Since labels were only used for the month change buttons, I think we can remove!